Module 7/ex0/Includes/Whatever

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+template <typename T> class Whatever {
+public:
+  Whatever() = delete;
+  ~Whatever() = default;
+
+private:
+  T _value1, _value2, _min1, _min2, _max1, _max2;
+};
+template <typename T> void swap(T &value1, T &value2);
+template <typename T> T min(const T &min1, const T &min2);
+template <typename T> T max(const T &max1, const T &max2);
+
+template <typename T> void swap(T &value1, T &value2) {

```

Comment:
To highlight the template part, I would do this in 2 lines:
template <typename T>
void swap (...)